tools/breaking-change-detector/GNUmakefile

```diff
@@ -12,11 +12,14 @@ endif
 
 clean:
 	cd old/; \
```

Comment:
What happens if these folders don't exist? will the command still succeed?

Reply:
no - you need to run `make clone` first (or set them up some other way.) See [README changes](https://github.com/GoogleCloudPlatform/magic-modules/blob/a791b3637bd77f153a8a5f83d6c518e8b8f190e4/tools/breaking-change-detector/README.md) for more details

Reply:
happy to address in a follow-on PR if you have concerns with this approach